back/src/Model/Group.ts

```diff
@@ -59,26 +79,27 @@ export class Group implements Movable {
         };
     }
 
```

Comment:
Is this suggested comment correct?

```suggestion
/**
 * Returns the list of users of the group, ignoring any "followers".
 * Useful to compute the position of the group if a follower is "trapped" far away from the the leader.
 */
```